### ISSUE TITLE
Release v0.14.0

### ISSUE THREAD

**Issue by ButterscotchV** [SlimeVR/SlimeVR-Server#1258]
Tracking release v0.14.0. 🐄
Estimated release: March-April

Needs Review(s)/Merge:

Merged:
- #1237
- #1240
- #1091
- #1227
- #1235
- #1244
- #1246
- #1242
- #1256
- #1254
- #1253
- #1261
- #880
- #1241
- #1269
- #1255
- #1277
- #1249
- #1272
- #1265
- #1276
- #1278
- #1262
- #1247
- #1287
- #1156
- #1222
- #1285
- #1290
- #1286
- #1299
- #1302
- #1300
- #1304
- #1303
- #1294
- #1317
- #1319
- #1301
- #1318
- #1322
- #1324
- #1310
- #1312
- #1284
- #1325
- #1331
- #1330
- #1338
- #1332
- (Direct commit, "Better tones") https://github.com/SlimeVR/SlimeVR-Server/commit/808dd0f6706a456912f1758bc5d703cffedaadc5
- #1339
- #1340
- #1342
- #1336
- #1348
- #1347
- #1351
- #1350
- #1329
- #1353
- #1357
- #1153
- #997
- #1363
- #1367
- #1368
- #1370
- #1372
- #1373
- #1371
- #1375

Known Issues:
- #1309
  - Workaround: Disable OSCQuery or set OSC port to something unusable and rely on OSCQuery
- Firmware updates are not staggered, problematic firmware will be pushed immediately and may cause issues.

RC Testing:
1. Android build does not crash.
   - #1240
2. VMC functions the same as previous releases.
   - #1091
3. VRM can be imported and removed.
   - #1246
4. Hips work as expected and do not have a tilt directly after a full reset.
   - #1244
5. AutoBone doesn't hit the error limit with intended usage (target ~0.012 error).
   - #1249
   - #1255
6. LINUX: AppImage doesn't crash.
   - #1253
7. Firmware tool is able to build and flash firmware on DIY trackers.
   - #880
8. Firmware tool is able to update firmware on official trackers successfully.
   - #1241
9. Serial functions the same as previous releases.
   - Should be fine, but firmware tool changes may have broken something? Make sure all serial functionality still works as expected.
10. Moving average quaternions don't bug over time, obvious with extended spine model enabled (waist sticks out backwards).
    - #1091
    - #1244
    - #1255
    - #1277
    - #1278
    - #1302
11. The "trackers still on" popup doesn't show up when closing via tray.
    - #1265
12. Logs function and are limited to only the latest 2 files (oldest ones are deleted).
    - #1262
13. The latest log's content is up-to-date with the current console output.
    - #1262
14. LINUX: SolarXR IPC socket functions as expected with Monado.
    - #1247
15. Skeleton height config allows measuring the headset height and floor height, validating that the height is above 0.4m.
    - #1156
16. Skeleton height config allows setting the height manually when no headset tracker is present, validating that the entered height is above 0.4m.
    - #1156
17. Skeleton height config loads previously configured values, validating that the configured height is above 0.4m.
    - #1156
18. Skeleton constraints function as intended, limiting the skeleton joints when enabled.
    - #1222
19. Skeleton constraints can be enabled and disabled.
    - #1290
20. Modified body proportions don't reset after tracker disconnect/reconnect.
    - #1304
21. New trackers are not assigned to the "Head" tracker position by default.
    - #1313
    - #1317
22. Toe snap and foot plant function.
    - #1350

**Comment by ButterscotchV** [comment]
> I would like to target it end of December. At least have a RC...
> 
> And I really want the new onboarding too.

We can try for December RC, I have a feeling we won't be able to get past RC for December though

**Comment by ImUrX** [comment]
Nah, i dont mind either of them, just less burden on having to edit the page when we merge imo

**Comment by ImUrX** [comment]
https://github.com/SlimeVR/SlimeVR-Server/pull/1234 requires merge of firmware first also, i think thats how we should merge firmware-dependent PRs for now

**Comment by FluffyOMC** [comment]
List needs updated for #1269 merge

**Comment by ButterscotchV** [comment]
v0.14.0 is released :3
